### PR TITLE
Helm deployment documentation

### PR DESCRIPTION
**Pull request by dejanb**

### CODE REVIEW

File: site/content/deployment/kubernetes.md [hunk 1]
```diff
@@ -45,6 +45,71 @@ From the directory `deploy/target/deploy/kubernetes` run:
...
+
+~~~sh
+~hono/deploy$ helm delete --purge eclipse-hono
+~hono/deploy$ kubectl delete crd prometheuses.monitoring.coreos.com
+~hono/deploy$ kubectl delete crd prometheusrules.monitoring.coreos.com
+~hono/deploy$ kubectl delete crd servicemonitors.monitoring.coreos.com
+~hono/deploy$ kubectl delete crd alertmanagers.monitoring.coreos.com
+~~~
+
+The additional delete commands are necessary to clean [Prometheus operator CRDs](https://github.com/helm/charts/tree/master/stable/prometheus-operator#uninstalling-the-chart).
```
Comment (sophokles73) [inline site/content/deployment/kubernetes.md]: Are the CRDs related to Prometheus only? Or do they belong to OLM?

Comment (dejanb) [reply site/content/deployment/kubernetes.md]: They are related to Prometheus operator only. OLM is not mandatory for the Helm deployment.

Comment (sophokles73) [reply site/content/deployment/kubernetes.md]: So, in this case you are deploying Prometheus *without* OLM, right?

Comment (dejanb) [reply site/content/deployment/kubernetes.md]: That is my understanding, yes. It's uses only Prometheus Operator for now. We can see how to approach all this in the future.

**Comment by sophokles73** [comment]
@dejanb can you also add a note to the release notes about the new possibility to deploy?

**Comment by dejanb** [comment]
Done

**Pull request merged**